campsite-booking-service/pom.xml

```diff
@@ -70,6 +70,10 @@
             <groupId>org.apache.derby</groupId>
             <artifactId>derby</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@kilyinov, please add this depenedcy to the `dependencyManaegement` in the parent pom.xml.